src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,7 @@
-export const UserInfo = () => <>Put the user here</>;
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
```

Comment:
```suggestion
export const UserInfo = ({ user: { name, email } }) => (
```
You can do destructuring assignment for nested objects in this way
Please, do it (or do the destructuring assignment separately) in all similar cases